samples/ConPTY/EchoCon/EchoCon/EchoCon.cpp

```diff
@@ -182,16 +167,26 @@ HRESULT InitializeStartupInfoAttachedToPseudoConsole(STARTUPINFOEX* pStartupInfo
     return hr;
 }
 
-COORD GetConsoleSize(HANDLE hStdOut)
+void __cdecl PipeListener(LPVOID pipe)
 {
-    COORD size = { 0, 0 };
+    HANDLE hPipe{ pipe };
+    HANDLE hConsole{ GetStdHandle(STD_OUTPUT_HANDLE) };
+
+    const DWORD BUFF_SIZE{ 512 };
+    char szBuffer[BUFF_SIZE]{};
```

Comment:
consider this:

```
char szBuffer[512]{};
…
ReadFile(hPipe, szBuffer, std::extent<decltype(szBuffer)::value, …)
```

to kill a silly YELLING_CASE constant

Reply:
Why use 9 chars when a namespaced templated expression will suffice. Doesn't add anything of significant value and just complicates things for the casual observer.